specification/elastic/resource-manager/Microsoft.Elastic/preview/2023-02-01-preview/elastic.json

```diff
@@ -1755,6 +1798,10 @@
           "format": "int32",
           "type": "integer",
           "readOnly": true
+        },
+        "generateApiKey": {
```

Comment:
>generateApiKey [](http://example.com/codeflow?start=9&length=14)

consider making it an enum

Reply:
It is just a true/false value hence boolean seems more appropriate.